### ISSUE TITLE
Resnet Model always predicting same label

### ISSUE THREAD

**Issue by adam-mahmood** [pytorch/pytorch#38009]
I am currently playing around with PyTorch trying to successfully attempt facial recognition on a custom dataset of 48 classes with over 5000 images using Resnet50. During training my training and validation set accuracies are fairly high in the regions of 99% for both sets, however when I attempt to test my model and a number of unseen test images, I always get the same label? Would you know what the reason for this is?Is this a bug?

**Comment by ptrblck** [comment]
Hi @adam-mahmood,

would you mind creating a topic in the [forum](https://discuss.pytorch.org/) so that we could discuss your use case there?
This would keep the GitHub Issues to bugs and feature requests.

**Comment by adam-mahmood** [comment]
Hi @ptrblck 

Thank you for your reply. I have created a topic and I would appreciate it if you could take a look at
[](https://discuss.pytorch.org/t/resnet-model-always-predicting-same-label/80669?u=adam_mahmood)
